adapter/storage/src/main/java/org/apache/rocketmq/eventbridge/adapter/storage/rocketmq/runtimer/RocketMQEventSubscriber.java

```diff
@@ -185,11 +196,19 @@ private void initMqProperties() {
             String socks5Password = properties.getProperty("rocketmq.consumer.socks5Password");
             String socks5Endpoint = properties.getProperty("rocketmq.consumer.socks5Endpoint");
 
+            String metricsPromExporterHost = properties.getProperty("metrics.endpoint.host");
+            String metricsPromExporterPort =  properties.getProperty("metrics.endpoint.port");
+            String metricsCollectorMode = properties.getProperty("metrics.collector.mode");
             clientConfig.setNameSrvAddr(namesrvAddr);
             clientConfig.setAccessChannel(AccessChannel.CLOUD.name().equals(accessChannel) ?
                     AccessChannel.CLOUD : AccessChannel.LOCAL);
             clientConfig.setNamespace(namespace);
+            BridgeConfig bridgeConfig = new BridgeConfig();
+            bridgeConfig.setMetricsPromExporterHost(metricsPromExporterHost);
+            bridgeConfig.setMetricsPromExporterPort(Integer.parseInt(metricsPromExporterPort));
```

Comment:
Why the configuration of metrics exporter is configured in RocketMQEventSubscriber instead of EventMetricsManager?